src/main/java/org/alfresco/repo/copy/CopyServiceImpl.java

```diff
@@ -313,7 +314,7 @@ public NodeRef copyAndRename(
 
         while (this.internalNodeService.getChildByName(destinationParent, assocTypeQName, newName) != null)
         {
-            newName = I18NUtil.getMessage(COPY_OF_LABEL, newName);                        
+            newName = buildNewName(destinationParent, assocTypeQName, newName);

```

Comment:
You should remove the redundant parameters destinationParent and assocTypeQName.